packages/calcite-components/src/components/accordion-item/accordion-item.scss

```diff
@@ -1,3 +1,32 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden
+ *
+ * @prop --calcite-accordion-item-background: Specifies the background color of the accordion item.
+ * @prop --calcite-accordion-item-space: Specifies the base spacing multiplier for padding and margin of the accordion item.
```

Comment:
Do we want to expose `-space` and `-size` here? The rest look good IMO to be public.